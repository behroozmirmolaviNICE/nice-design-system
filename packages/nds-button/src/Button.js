// @flow
import React, { Component } from "react";

type ButtonProps = {
	id: ?string,
	title: string
};

export default class Button extends Component<ButtonProps> {
	render() {
		return <button className="btn">

		</button>;
	}
}
