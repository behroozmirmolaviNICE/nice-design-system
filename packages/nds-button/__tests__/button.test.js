"use strict";

import React from "react";
import { mount } from "enzyme";

import Button from "../src/Button";

describe("@nice-digital/nds-button", () => {

	let wrapper;

	it("renders without crashing", () => {
		wrapper = mount(<Button>Example</Button>);
	});

	it("should render a button", () => {
		expect(wrapper.find("button")).toBeTruthy();
	});

	it("matches snapshot", () => {
		expect(wrapper).toMatchSnapshot();
	});
});
