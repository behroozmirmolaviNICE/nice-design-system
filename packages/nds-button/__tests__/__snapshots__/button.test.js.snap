// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@nice-digital/nds-button matches snapshot 1`] = `
<Button
  type="button"
>
  <button
    className="btn"
    type="button"
  >
    Example
  </button>
</Button>
`;
